python/labbox_ephys/misc/get_timeseries_segment.py

```diff
@@ -40,15 +41,18 @@ def get_timeseries_segment(recording_object, ds_factor, segment_num, segment_siz
         traces_reshaped = traces.reshape((M, N2, ds_factor))
         traces_min = np.min(traces_reshaped, axis=2)
         traces_max = np.max(traces_reshaped, axis=2)
-        traces = np.zeros((M, N2 * 2))
+        traces = np.zeros((M, N2 * 2), dtype=np.float32)
         traces[:, 0::2] = traces_min
         traces[:, 1::2] = traces_max
     
-    data_b64 = _mda32_to_base64(traces)
-    # elapsed = time.time() - timer
-    return dict(
-        data_b64=data_b64
-    )
+    return {
```

Comment:
We can now directly return numpy arrays from the hither functions, as the labbox framework takes care of the serialization